sys/linux/sys.txt

```diff
@@ -494,6 +495,9 @@ pidfd_send_signal(fd fd_pidfd, sig signalno, info ptr[in, siginfo], flags const[
 pidfd_open(pid pid, flags const[0]) fd_pidfd
 pidfd_getfd(pidfd fd_pidfd, fd fd, flags const[0]) fd
 
+close_range(fd fd, max_fd fd, flags const[CLOSE_RANGE_UNSHARE])
```

Comment:
The actual code considers `CLOSE_RANGE_UNSHARE` to be an optional flag, not required.  So it shouldn't be a `const`.

Reply:
I've just changed it to a flag. But from the source code it seems that any flag other than a CLOSE_RANGE_UNSHARE would result in -EINVAL. Why do we still need it to be a flag?

Reply:
It can be 0 and not result in `EINVAL`, right?  Whereas a const implies the flag always must be specified.

Reply:
Now it's `flags flags[close_range_flags]`, which looks correct.  @JiahengHu please feel free to resolve comment threads yourself -- thanks!